build_wasm.sh

```diff
@@ -0,0 +1,44 @@
+PROJECT_NAME=weblibdedx
+WASM_PUBLIC=./public/$PROJECT_NAME.wasm
+PROPER_PATH="\/web_dev\/$PROJECT_NAME.wasm"
+JS=./src/Backend/$PROJECT_NAME.js
+WASM_LOOKUP="wasmBinaryFile = locateFile"
+
+cd ./libdedx
+
+git checkout 769af2d9fd94b15c8ea08a973f94bdfdf048a82f
```

Comment:
What is that ?

Reply:
It's a checkout for the subodule. We need to complie the version from Piotr's branch, since the functions on master still return const int *

Reply:
Can't we simply ask him to merge it quickly ? I just had few remarks.
@piotrpolec what do you think ?

Reply:
We'd need to send a working demo to prof. Szydło. It would be best done by today and i don't think this is tested well enough to merge it to the master libdedx. What's so bad about this change? It's only a temporary solution to a problem we won't have in the future.

Reply:
OK, lets keep it, but we will add an issue to remove it later